src/Microsoft.ML.TensorFlow/TensorFlow/TensorGeneric.tt

```diff
@@ -37,13 +37,14 @@ namespace Microsoft.ML.Transforms.TensorFlow
         /// </summary>
         /// <typeparam name="T[]">.NET type of tensor to create</typeparam>
         /// <param name="data">value of tensor</param>
+        /// <param name="count">The number of elements in the tensor</param>
```

Comment:
did we have to re-generate the TensorGeneric.cs file after making these changes ?

Reply:
Yes. Apparently TensorGeneric.tt regenerates TensorGeneric.cs automatically whenever it is saved.

---
In reply to: [216466572](https://github.com/dotnet/machinelearning/pull/857#discussion_r216466572) [](ancestors = 216466572)

Reply:
awesome. thanks for the info.

---
In reply to: [216479096](https://github.com/dotnet/machinelearning/pull/857#discussion_r216479096) [](ancestors = 216479096,216466572)